### PR TITLE
fix condition for KubePortManagerSingleton init twice

### PR DESCRIPTION
**Pull request by goodseeyou**
## What
Fix condition to init KubePortManagerSingleton twice when given ports are the same.

## How
make the condition align to the log message

## Recommended reading order
`airbyte-workers/src/main/java/io/airbyte/workers/process/KubePortManagerSingleton.java`

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Can you sign the CLA @goodseeyou ?

**Comment by marcosmarxm** [comment]
@davinchia can you check this contribution?

**Comment by goodseeyou** [comment]
Hi @marcosmarxm I signed it

**Comment by davinchia** [comment]
@goodseeyou can you explain the error case this logic change is catching? I want to make sure I understand what you are thinking.

We have pretty good test today in `KubePodProcessIntegrationTest`, can you add a test case there testing the situation you have in mind?

File: airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java [hunk 1]
```diff
@@ -121,6 +119,31 @@ public void teardown() throws Exception {
     server.stop();
   }
 
+  @RetryingTest(3)
+  public void testInitKubePortManagerSingletonTwice() throws Exception {
```
Comment (goodseeyou) [inline airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java:123]: @davinchia what I really add test here.

Others code change is coding-style formatter by google-style which recommends (https://github.com/airbytehq/airbyte/blob/master/docs/contributing-to-airbyte/code-style.md)

File: airbyte-workers/src/main/java/io/airbyte/workers/process/KubePortManagerSingleton.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ public static synchronized KubePortManagerSingleton getInstance() {
   public static synchronized void init(final Set<Integer> ports) {
     if (instance == null) {
       instance = new KubePortManagerSingleton(ports);
-    } else if (Sets.intersection(instance.getAllPorts(), ports).size() != ports.size()) {
```
Comment (goodseeyou) [inline airbyte-workers/src/main/java/io/airbyte/workers/process/KubePortManagerSingleton.java:59]: here is what I really fix

**Comment by goodseeyou** [comment]
local test successfully
Screen Shot 2022-09-16 at 00 22 38

**Comment by marcosmarxm** [comment]
@davinchia can you check this PR?

**Comment by davinchia** [comment]
@goodseeyou thank you!

File: airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java [hunk 3]
```diff
@@ -102,14 +103,31 @@ public void setup() throws Exception {
...
-            heartbeatUrl,
-            getHost(),
-            false);
+    processFactory = new KubeProcessFactory(workerConfigs, "default", fabricClient, heartbeatUrl, getHost(), false);
+  }
+
+  @RetryingTest(3)
+  public void testInitKubePortManagerSingletonTwice() throws Exception {
+    /**
+     * Test init KubePortManagerSingleton twice: 1. with same ports shoule succeed 2. with different
```
Comment (davinchia) [inline airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java:112]:
```suggestion
     * Test init KubePortManagerSingleton twice: 1. with same ports should succeed 2. with different
```

File: airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java [hunk 4]
```diff
@@ -102,14 +103,31 @@ public void setup() throws Exception {
...
+     * port should fail
+     *
+     * Every test has been init firt times in BeforeAll with getOpenPorts(30)
```
Comment (davinchia) [inline airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java:115]:
```suggestion
     * Every test has been init once in BeforeAll with getOpenPorts(30)
```

**Review by davinchia — APPROVED** [review]
@goodseeyou two small spelling suggestions. After that, good to merge!

**Comment by goodseeyou** [comment]
Hi @davinchia, I fixed the spelling by your suggestions.

**Comment by marcosmarxm** [comment]
Sorry the delay here @goodseeyou I'll merge this on Monday.

**Review by marcosmarxm — APPROVED** [review]
It is building locally! Thanks @goodseeyou sorry the delay here.

**Pull request merged**